gui/src/main/java/org/jboss/as/console/client/shared/subsys/undertow/HttpMetricPresenter.java

```diff
@@ -178,7 +179,7 @@ public void loadDetails() {
 
         ModelNode operation = new ModelNode();
         operation.get(OP).set(READ_RESOURCE_OPERATION);
-        operation.get(ADDRESS).set(Baseadress.get());
+        operation.get(ADDRESS).set(new ResourceAddress().add("host", serverStore.getSelectedServer().getHostName()).add("server", serverStore.getSelectedServer().getServerName()));
```

Comment:
@hpehl, you were right. It wasn't working for standalone. This should be fixed now.